### PR TITLE
Futurize pass 1

### PR DESCRIPTION
**Pull request by ydirson**
This includes:
- the pass 1 run itself
- fixing things futurize breaks
- avoiding the use of `future` in favor of `six` (`future` should not have appeared it anyway, as I read the definition of pass 1)
- cleanup changes from with pylint help

### CODE REVIEW

File: tests/test_pci.py [hunk 1]
```diff
@@ -9,7 +9,7 @@ class TestInvalid(unittest.TestCase):
     def test_invalid_types(self):
 
         self.assertRaises(TypeError, PCI, 0)
-        self.assertRaises(TypeError, PCI, 0L)
+        self.assertRaises(TypeError, PCI, 0)
```
Comment (stormi) [inline tests/test_pci.py:12]: You've got twice the same test now

File: xcp/bootloader.py [hunk 2]
```diff
@@ -432,7 +432,7 @@ def create_label(title):
             # If this fails, it is probably a string, so leave it unchanged.
             try:
                 default = menu_order[int(default)]
-            except ValueError as KeyError:
+            except (ValueError, KeyError):
```
Comment (stormi) [inline xcp/bootloader.py:435]: IMO it's not futurize being wrong here, it's the code being buggy initially due to missing parentheses (KeyError was never catched as far as I can tell, as the following test I just did shows).

```
>>> try:
...     raise MonException, "blabla lalili"
... except IOError, MonException:
...     print "Catched it"
... 
Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
__main__.MonException: blabla lalili
>>>
```

File: xcp/bootloader.py [hunk 3]
```diff
@@ -525,7 +526,8 @@ def writeGrub(self, dst_file = None):
             m = self.menu[label]
             print("\ntitle " + m.title, file=fh)
             if m.hypervisor:
-                print("   kernel " + m.hypervisor + " " + m.hypervisor_args, file=fh)
+                print("   kernel " + m.hypervisor + " " + m.hypervisor_args,
```
Comment (stormi) [inline xcp/bootloader.py:529]: Didn't you allow up to 100 chars per line, previously?

File: xcp/bootloader.py (hunk 2 above)
Comment (ydirson) [reply xcp/bootloader.py:435]: right, it warrants a pre-futurize fix instead

File: xcp/bootloader.py (hunk 3 above)
Comment (ydirson) [reply xcp/bootloader.py:529]: many commits were written before we decided to switch to 100, sure some line wraps are not necessary any more now

File: xcp/bootloader.py (hunk 2 above)
Comment (stormi) [reply xcp/bootloader.py:435]: Funny, now github shows my (outdated) comment here.

File: tests/test_pci.py (hunk 1 above)
Comment (ydirson) [reply tests/test_pci.py]: addressed

File: xcp/bootloader.py (hunk 3 above)
Comment (ydirson) [reply xcp/bootloader.py]: unwrapped

**Review by psafont — APPROVED** [review]

**Comment by edwintorok** [comment]
This is a stacked PR, compare commits for just this PR: https://github.com/ydirson/xenserver-python-libs/compare/improve-test-coverage...ydirson:xenserver-python-libs:futurize-pass-1

**Comment by edwintorok** [comment]
next https://github.com/ydirson/xenserver-python-libs/compare/futurize-pass-1...ydirson:xenserver-python-libs:futurize-pass-2

**Review by xennifer — APPROVED** [review]

**Pull request merged**